docs/source/science/ESA_CCI/ESA_CCI_V4.ipynb

```diff
@@ -17,52 +17,39 @@
     "tags": []
```

Comment:
<p>The naming of the variables is confusing, as <code>min_y</code> is actually <code>-5.0</code> but we swapped it because of how the order of the <code>y</code> dimension coordinates. I would rewrite this as follows (correct assignments, but swapped in <code>y=slice(...)</code>)</p><p> </p><pre class="ql-syntax" spellcheck="false">min_x = -55
max_y = -3.0
max_x = -50.0
min_y = -5.0

arr_sub = arr.sel(x=slice(min_x, max_x), y=slice(max_y, min_y))
arr_sub
</pre><p> </p><p>That said, I would suggest to reuse the first bbox (see my comment at the beginning of the notebook)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Good point. Thanks</p>